client-static/src/main/kotlin/app/cash/backfila/client/static/internal/StaticDatasourceBackfillOperator.kt

```diff
@@ -27,29 +25,18 @@ class StaticDatasourceBackfillOperator<I : Any, P : Any>(
       parametersOperator.constructBackfillConfig(request.parameters, request.dry_run)
     backfill.validate(config)
 
-    val start: Int
-    val end: Int
-    try {
-      start = request.range?.start?.utf8()?.toInt() ?: 0
-    } catch (e: NumberFormatException) {
-      throw BadRequestException("Start of range must be a number", e)
-    }
-
-    try {
-      end = request.range?.end?.utf8()?.toInt() ?: backfill.staticDatasource.size
-    } catch (e: NumberFormatException) {
-      throw BadRequestException("End of range must be a number", e)
-    }
+    val start = request.range?.start?.utf8()?.toInt() ?: 0
+    val end = request.range?.end?.utf8()?.toInt() ?: backfill.staticDatasource.size
```

Comment:
Do we want to use `toIntOrNull()` here instead of `toInt()`? That would take care of possible parse errors.

Gives us the option to just use defaults if we can't parse a number, or do something like the following without losing the messaging that would otherwise be propagated to the caller.

```kt
val startString = request.range?.start?.utf8() ?: "0"
val start = start.toIntOrNull() ?: error("Start of range must be a number")
```